### ISSUE TITLE
benchmarking with WRK I'm getting Non-2xx responses after add jackson support

### ISSUE THREAD

**Issue by eriknyk** [ktorio/ktor#372]
Hi, when I run a very basic example:

```
fun main(args: Array<String>) {
    val server = embeddedServer(Netty, port = 9000) {
        routing {
            get("/hello") {
                call.respondText("Hello")
            }
        }
    }
    server.start(wait = true)
}
```
It work fine and I got the following WRK results:

![image](https://user-images.githubusercontent.com/205721/38002516-6d17e39c-31f8-11e8-9bb1-03453df1baf6.png)

But when I enable json support using jackson() or even gson() I got same result with Non-2xx results, 

```
fun main(args: Array<String>) {
    val server = embeddedServer(Netty, port = 9000) {
        install(ContentNegotiation) {
            jackson {
                configure(SerializationFeature.INDENT_OUTPUT, true)
                registerModule(JavaTimeModule())
            }
        }

        routing {
            get("/hello") {
                val person = Person("J", "John")

                call.respond(person)
            }
        }
    }
    server.start(wait = true)
}
```

basically all requests failed:

![image](https://user-images.githubusercontent.com/205721/38002429-ed01e13a-31f7-11e8-97d5-82c8f20d9e38.png)

I will appreciate any help, 

Best Regards.

**Comment by soywiz** [comment]
I have verified that this happens. Using wireshark, this is the HTTP call with WRK:

```
GET /hello HTTP/1.1
Host: 127.0.0.1:9000

HTTP/1.1 406 Not Acceptable
Content-Length: 0
```

Probably because no Accept header. With Chrome for example, this is the call:

```
GET /hello HTTP/1.1
Host: 127.0.0.1:9000
Connection: keep-alive
Pragma: no-cache
Cache-Control: no-cache
Upgrade-Insecure-Requests: 1
User-Agent: Mozilla/5.0 ...
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8
Accept-Encoding: gzip, deflate, br
Accept-Language: es,en-US;q=0.9,en;q=0.8

HTTP/1.1 200 OK
Content-Length: 40
Content-Type: application/json; charset=UTF-8
Connection: keep-alive

{
  "name" : "J",
  "surname" : "John"
}
```

Chrome sends
`Accept: */*;q=0.8`
That accepts everything as fallback. Maybe we could asume a `*/*` when no Accept header is sent?

**Comment by soywiz** [comment]
This was fixed already here: https://github.com/ktorio/ktor/issues/317

Try to use ktor version `0.9.2-alpha-3`. (It should be fixed since `0.9.2-alpha-1`).
I have verified that it fails on 0.9.1 but works on 0.9.2-alpha-3.

**Comment by eriknyk** [comment]
using `0.9.2-alpha-3` it works normally, gracias @soywiz
